docs/UserGuide.md

```diff
@@ -1,186 +1,85 @@
-# AddressBook Level 4 - User Guide
+# User Guide
 
-By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbsp;&nbsp; Licence: `MIT`
+This product is not meant for end-users and therefore there is no user-friendly installer. 
+Please refer to the [Setting up](DeveloperGuide.md#setting-up) section to learn how to set up the project.
 
----
+## Starting the program
 
-1. [Quick Start](#quick-start)
-2. [Features](#features)
-3. [FAQ](#faq)
-4. [Command Summary](#command-summary)
-
-## 1. Quick Start
-
-0. Ensure you have Java version `1.8.0_60` or later installed in your Computer.<br>
-
-   > Having any Java 8 version is not enough. <br>
-   > This app will not work with earlier versions of Java 8.
-
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
-3. Double-click the file to start the app. The GUI should appear in a few seconds.
-   > <img src="images/Ui.png" width="600">
-
-4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
-   e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window.
-5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` :
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
-   * **`exit`** : exits the app
-6. Refer to the [Features](#features) section below for details of each command.<br>
-
-
-## 2. Features
-
-> **Command Format**
->
-> * Words in `UPPER_CASE` are the parameters.
-> * Items in `SQUARE_BRACKETS` are optional.
-> * Items with `...` after them can have multiple instances.
-> * Parameters can be in any order.
-
-### 2.1. Viewing help : `help`
+1. Find the project in the `Project Explorer` or `Package Explorer` (usually located at the left side)
+2. Right click on the project
+3. Click `Run As` > `Java Application` and choose the `Main` class.
+4. The GUI should appear in a few seconds.
 
+## Viewing help : `help`
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
-
-### 2.2. Adding a person: `add`
-
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
-
-> Persons can have any number of tags (including 0)
-
-Examples:
-
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
-
-### 2.3. Listing all persons : `list`
-
-Shows a list of all persons in the address book.<br>
+ 
+## Adding a task : `add`
+Adds a task to the task manager<br>
+Format: `add TASKNAME DATE [STARTTIME] [ENDTIME] VENUE DESCRIPTION` 
```

Comment:
Think about the user experience. Does user need to remember the order of parameters? How the user will know which one is STARTTIME, which one is ENDTIME and etc.